xml/cha_introduction.xml

```diff
@@ -296,4 +296,51 @@
   </important>
  </sect1>
 
+ <sect1 xml:id="sec-intro-support">
+  <title>Support</title>
+  <para>
+   Regardless of the public cloud you use and the plan you choose, &suse; has
+   you covered with support. Details however differ:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     PAYG instances include support through the cloud provider which in turn is
```

Comment:
"PAYG instances include support through the cloud provider." That's probably all we should say. SUSE provides L3 support for the CSP, but that's transparent to the customer.

Reply:
This is not going to be true for very much longer we are going to list images that are PAYG and are sold by SUSE directly. For those images we provide L1, L2, and L3 support. I would not state it that way.

Reply:
Is it? I would not understand it without the "which in turn…" clause.

Also, that wording is already used in the 'Plans' section. It originally comes from Robert in ba58c7e42. Therefor I suggest to keep it.